es6-persistence/src/main/java/com/netflix/conductor/dao/es6/index/ElasticSearchDAOV6.java

```diff
@@ -156,8 +139,8 @@ public ElasticSearchDAOV6(Client elasticSearchClient, ElasticSearchConfiguration
         this.objectMapper = objectMapper;
         this.elasticSearchClient = elasticSearchClient;
         this.indexPrefix = config.getIndexName();
-        this.workflowIndexName = indexName(WORKFLOW_DOC_TYPE);
-        this.taskIndexName = indexName(TASK_DOC_TYPE);
+        this.workflowIndexName = getIndexName(WORKFLOW_DOC_TYPE);
```

Comment:
I suppose there're some legacy reasons behind this, but, shouldn't this doc types be an enum instead of a series of Strings?

Reply:
will refactor in a subsequent PR.